content/blog/2019/multicluster-version-routing/index.md

```diff
@@ -36,7 +36,7 @@ running in one cluster, versions `v2` and `v3` running in a second cluster.
 To start, you'll need two Kubernetes clusters, both running a slightly customized configuration of Istio.
 
 * Set up a multicluster environment with two Istio clusters by following the
-    [multiple control planes with gateways](/docs/setup/kubernetes/multicluster-install/gateways/) instructions.
+    [multiple control planes with gateways](/docs/setup/kubernetes/multicluster/gateways/) instructions.
```

Comment:
not sure this is right.  We will have install, tasks, and examples.

Reply:
Yes, but the non-install related content will not be published under the /setup folders.
This is a common problem, it's natural to want to add "install" to the folder or file name but the folder structure makes it unnecessary.